meos/include/meos.h

```diff
@@ -38,6 +38,9 @@
 #include <stdbool.h>
 #include <stdint.h>
 #include <stdio.h>
+/* GSL */
+#include <gsl/gsl_rng.h>
+#include <gsl/gsl_randist.h>
```

Comment:
Could you also remove the includes that were added in `meos.h`?
After that everything looks god so I'll accept and merge the PR.